src/picknik_ur_gazebo_scan_and_plan_config/description/simulation_worlds/scan_and_plan_world.sdf

```diff
@@ -0,0 +1,230 @@
+<sdf version="1.6">
+  <world name="space_station_apriltag_world">
```

Comment:
rename this world? Though clearly it makes no difference

Reply:
Done.